catalyst/contrib/data/nlp/language_modeling/tests/test_dataset.py

```diff
@@ -0,0 +1,24 @@
+import torch  # noqa: F401
+from transformers import AutoTokenizer
+
+from catalyst.contrib.data.nlp.language_modeling import LanguageModelingDataset
+
+texts = [
+    """Bonaparte Crossing the Alps is an oil-on-canvas painting by French artist""",  # noqa: E501
+    """Bhaskara's Lemma is an identity used as a lemma during the chakravala method. """,  # noqa: E501
+]
+
+
+def test_tokenizer_str():
+    """Test initialization with string"""
+    dataset = LanguageModelingDataset(texts, "bert-base-uncased")
+    assert dataset[0] is not None
+    assert len(dataset) == 2
+
+
+def test_tokenizer_tokenizer():
```

Comment:
btw, could we also add test for `exception raise`?

Reply:
done